src/Tribe/Customizer/Text.php

```diff
@@ -0,0 +1,272 @@
+				.tribe-events .tribe-events-calendar-month__calendar-event-tooltip-datetime {
+					color: <%= text.primary_text_color %>;
+				}
+			';
+			
+			$primary_rgb = $primary_text_color_rgb['R'] . ',' . $primary_text_color_rgb['G'] . ',' . $primary_text_color_rgb['B'];
+
+			$template .= '
+				.tribe-events .datepicker .datepicker-switch:hover, 
+				.tribe-events .datepicker .datepicker-switch:focus,
+				.tribe-events .tribe-events-c-view-selector__list-item-link:hover .tribe-events-c-view-selector__list-item-text, 
+				.tribe-events .tribe-events-c-view-selector__list-item-link:focus .tribe-events-c-view-selector__list-item-text {
+					color: rgba( ' . $primary_rgb . ', 0.8 );
+				}
+			';
+			
+			// PRO Styles.
+			$template .= '
+				.tribe-events-pro .tribe-events-pro-week-grid__events-time-tag,
+				.tribe-events-pro .tribe-events-pro-week-grid__event-tooltip-datetime,
+				.tribe-events-pro .tribe-events-pro-week-grid__header-column--current .tribe-events-pro-week-grid__header-column-weekday {
+					color: <%= text.primary_text_color %>;
+				}
+			';
+		
+			$template .= '
+				.tribe-events-pro.tribe-events-view--week .datepicker .day.disabled {
+					color: <%= text.primary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-events-pro.tribe-events-view--week .datepicker .day.past {
+					color: rgba( ' . $primary_rgb . ', 0.62 );
+				}
+			';
+
+		}
+
+		if ( $customizer->has_option( $this->ID, 'secondary_text_color' ) ) {
+			$template .= '
+				.tribe-common .tribe-common-form-control-toggle__label,
+				.tribe-events .tribe-events-calendar-list__event-date-tag-weekday {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border, 
+				.tribe-common a.tribe-common-c-btn-border {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-form-control-text__input::placeholder {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common--breakpoint-medium.tribe-events .tribe-events-c-nav__prev, 
+				.tribe-common--breakpoint-medium.tribe-events .tribe-events-c-nav__next {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			// PRO Styles.
+			$template .= '
+				.tribe-events .tribe-events-calendar-month__calendar-event-datetime,
+				.tribe-events-pro .tribe-events-pro-photo__event-date-tag-month,
+				.tribe-events-pro .tribe-events-pro-week-grid__event-datetime,
+				.tribe-events-pro .tribe-events-pro-map__event-date-tag-month,
+				.tribe-common--breakpoint-medium.tribe-events-pro .tribe-events-pro-map__event-distance {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+			
+		}
+
+		return $template;
+	}
+
+	/**
+	 * Setup the Customizer section.
+	 * 
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function setup() {
+		$this->defaults = [
+			'primary_text_color'   => '#141827',
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 79dbc8d093cd32cd8698e4d2fab4454f</sub>